### DISCUSSION TITLE
CAN communication problem with Medianav MN3 (evolution2)

### DISCUSSION THREAD

**Discussion by alex-military** [cedricp/ddt4all#1327] [Q&A]
hello I have a sandero stepway 2012 to which I installed a medianav mn3 with a dorid auto. having an error in the car's ecu related to the infotainment (probably related to the different registered chassis number) and having to change the settings of the svrc of the rear camera and the height of the guidelines I need to access it via ecutweaker, I have an ecu.zip file dedicated to dacias with all types of medianav from 1 to 4 I purchased a recommended wofi odb tested and working with other control units, I purchased a Y splitter from aliexpress on one of the 2 connectors I performed the cable swap on the first attempt I must have done something wrong because it gave me a can error, I checked the pins again and tried again and it says no data, what could it be due to? am I doing something wrong in crossing the cables? am I doing something wrong? even entering any setting it does not read anything at all no parameters how can I solve it?

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/970
